rapid7/metasploit-framework
# Add tests for aux and exploit cmd_check and cmd_run

Adds tests for the cmd_check and cmd_run functionality for auxiliary/exploit dispatchers

These might be considered brittle tests as they assert on exact string values that are logged to the console, and they might change in the future. However, I'm creating these tests as a pre-requisite to landing [smb/http uri support in rhosts](https://github.com/rapid7/metasploit-framework/pull/15253), so the semantic differences are clear

## Verification

- Ensure CI passes
- Ensure there's no important tests missing
- Run an auxiliary module
```
use auxiliary/scanner/http/title
set rhosts example.com
run
```
